index.js

```diff
@@ -10,31 +10,41 @@ import { getImageHandler, imageGenerationHandler } from './routes/image';
 const { preflight, corsify } = createCors();
 
 // Create a new router
-const router = Router();
+const router = Router({ base: '/v1' });
```

Comment:
The base should be configurable via environment variable and the default should be blank, as in no added base URL.